multi-node/aws/pkg/config/config_test.go

```diff
@@ -253,3 +253,59 @@ func TestIsSubdomain(t *testing.T) {
 	}
 
 }
+
+func TestReleaseChannel(t *testing.T) {
+
+	validConfigs := []struct {
+		conf    string
+		channel string
+	}{
+		{
+			conf: `
+releaseChannel: alpha
+`,
+			channel: "alpha",
+		},
+		{
+			conf: `
+releaseChannel: beta
+`,
+			channel: "beta",
+		},
+	}
+
+	invalidConfigs := []string{
+		`
+#TODO(chom): move this to validConfigs when stable is supported
```

Comment:
Seems weird to have the TODO as part of the test string. Also, as a nit, do these need to be multi-line strings. Can they not just be `[]string{"foo"}` instead?